packages/client/src/forms/register/mappings/query/field-mappings.ts

```diff
@@ -276,47 +285,30 @@ export const addressLineToFieldTransformer =
       queryData[sectionId] &&
       queryData[sectionId].address &&
       queryData[sectionId].address.find(
-        (addr: Address) => addr.type === addressType
+        (addr: Address) => addr.type === config.useCase
       )
 
     if (!address) {
       return transformedData
     }
 
-    transformedData[sectionId][field.name] =
-      (address.line && address.line[lineNumber]) || ''
+    if (config.lineNumber) {
```

Comment:
This will be skipped if `lineNumber == 0`

Reply:
Yes, thats right.  In the query mapping functions that is actually the intended result weirdly.